CoreMLPredictionsPlugin.podspec

```diff
@@ -1,4 +1,9 @@
-load 'build-support/dependencies.rb'
+# Version definitions
+$AMPLIFY_VERSION = '1.0.1'
+$AMPLIFY_RELEASE_TAG = "v#{$AMPLIFY_VERSION}"
+
+$AWS_SDK_VERSION = '2.13.4'
+$OPTIMISTIC_AWS_SDK_VERSION = "~> #{$AWS_SDK_VERSION}"
```

Comment:
Remove, since AWS version isn't used in this podspec

Reply:
nice catch, thanks